contrib/pyzfs/libzfs_core/__init__.py

```diff
@@ -38,6 +38,7 @@
 
     Maximum length of any ZFS name.
 '''
+from __future__ import absolute_import, division, print_function
```

Comment:
@behlendorf How do you define 'a little while'?
Asking since python 2.6 is dead and buried >5 years now and RHEL/CentOS 6 is able to run 2.7
(https://snarky.ca/stop-using-python-2-6/ contains reasoning why to drop 2.6 that I agree with).

Reply:
I put a blanket import of these `__future__` imports in an effort to reduce variability in different situations. If we hit an integer vs. float division issue, I'd rather see it "fail-fast" as much as possible, rather than lurk as a hidden issue.

If we want a different policy, I'll change that. Just my reasoning here.

Reply:
> How do you define 'a little while'?

RHEL6 goes [EOL in November 2020](https://access.redhat.com/support/policy/updates/errata).  If we want to drop support for 2.6 we need to split all the python in to new optional python2/python3  packages.  Which I believe was generally already agreed to above, it just needs to be done.